inc/namespace.php

```diff
@@ -82,7 +82,10 @@ function create_tables() {
 		`schedule` varchar(255) DEFAULT NULL,
 
 		PRIMARY KEY (`id`),
-		KEY `status` (`status`)
+		KEY `status` (`status`),
+		KEY `site` (`site`),
+		KEY `hook_args` (`hook`, `args`(50)),
```

Comment:
As per @rmccue's comment lets just index `hook` on its own

Reply:
`args` removed from index in 3227c6b797f2661a9959a2e0c5076434885c85ef.

I've got the _allow edits from maintainers_ option checked on this PR so you should be able to push to my repo if needs be, presuming GH's permissions model is entirely bug free.